src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -190,6 +189,8 @@ export class InputTimePicker
 
   popoverEl: HTMLCalcitePopoverElement;
 
+  private dialogId = `time-picker-dialog--${guid()}`;
```

Comment:
Nitpick: can you move this above so that the private properties are in alphabetical order?